### ISSUE TITLE
Riak 1.1.2 RC3: fitting was done before startup

### ISSUE THREAD

**Issue by slfritchie** [basho/riak_pipe#50]
Riak 1.1.2 RC3, x86_64 CentOS 6.2.

```
21:06:26.819 [error] Pipe worker startup failed:fitting was gone before startup
21:06:26.855 [error] Supervisor riak_pipe_builder_sup had child undefined started with {riak_pipe_builder,start_link,undefined} at <0.27541.5> exit with reason no such process or port in call to [{erlang,link,[<0.27560.5>],[]},{riak_kv_pipe_index,queue_existing_pipe,4,[{file,[115,114,99,47,114,105,97,107,95,107,118,95,112,105,112,101,95,105,110,100,101,120,46,101,114,108]},{line,150}]},{riak_kv_mrc_pipe,'-send_inputs_async/3-fun-0-',3,[{file,[115,114,99,47,114,105,97,107,95,107,118,95,109,114,99,95,112,105,112,101,46,101,114,108]},{line,492}]}] in context child_terminated
21:06:26.877 [error] Pipe worker startup failed:fitting was gone before startup
21:06:26.941 [error] Pipe worker startup failed:fitting was gone before startup
21:06:27.085 [error] Unrecognized message {pipe_log,#Ref<0.0.8.19715>,index,{trace,[error],{vnode,{fitting_died,822094670998632891489572718402909198556462055424}}}}
```

**Comment by slfritchie** [comment]
See https://github.com/basho/riak_pipe/issues/49#issuecomment-7329745 for basho_bench config to drive this.

**Comment by beerriot** [comment]
I believe that #75 addresses the final message in this report (the others having been addressed by basho/riak_kv#416 and basho/riak_kv#377). Closing.
